DQM/HcalCommon/interface/ValueQuantity.h

```diff
@@ -368,12 +407,44 @@ namespace hcaldqm
 				int _n;
 		};
 
+		/**
+		 * Coarse LumiSection axis. Specify binning (default=10 LS)
+		 */
+		class LumiSectionCoarse : public ValueQuantity
+		{
+			public:
+				LumiSectionCoarse() : ValueQuantity(fLS), _n(4000), _binning(10)
+				{}
+				LumiSectionCoarse(int n, int binning) : ValueQuantity(fLS), 
+					_n(n), _binning(binning)
+				{}
+				~LumiSectionCoarse() override {}
+				
+				LumiSectionCoarse* makeCopy() override
```

Comment:
Is this ever used? If not, please remove it.

Reply:
Yes, the coarse lumisection axis is used for the new LS plots in LaserTask. See for example https://github.com/cms-sw/cmssw/pull/23005/files#diff-b6fbe956e587bf03b7db6aa2f6eeae42R248. Also, this axis will likely be used more commonly going forward to reduce the memory usage of LS histograms.